src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -1323,25 +1347,33 @@ GenTree* Lowering::LowerHWIntrinsic(GenTreeHWIntrinsic* node)
         if (BlockRange().TryGetUse(node, &use))
         {
             GenTree* user = use.User();
+
+            JITDUMP("lowering EmbeddedMasked HWIntrinisic (before):\n");
+            DISPTREERANGE(BlockRange(), node);
+            JITDUMP("\n");
+
             // Wrap the intrinsic in ConditionalSelect only if it is not already inside another ConditionalSelect
```

Comment:
This comment isn't valid anymore, we're always introducing the ConditionalSelect and allowing lowering to optimize it further if applicable